hig.js

```diff
@@ -87,17 +88,41 @@ function speak(phrases) {
 		return false;  	
 	}
 	var p = phrases.shift();
-	var v = 'Alex';
-	if (p.name == ANNOUNCER) {
-		v = 'Samantha';
-	} 
+	if(os == 'mac'){  //mac male voice to talk as hockey player
+		var v = 'Alex';
+			if (p.name == ANNOUNCER ) { //windows female voice to talk as announcer 
+				v = 'Samantha';
+		} 
+	}
+	// else if (os=='win32'){ //windows male voice to talk as hockey player
+		// var v = 'David';
+			// if (p.name == ANNOUNCER ) { //linux female voice to talk as announcer 
+				// v = 'Zira';
+		// }
+	// }
+	// else if (os == 'Darwin'){ // Linux Male voice to talk as hockey player
```

Comment:
darwin is the mac os, so this is inconsistent

Reply:
Yeah, my mistake. I will fix this and make sure to double check next time. Thanks.

Reply:
Fixed, sorry about that. Ill make sure to double check and be more diligent.